feature-libs/my-account/organization/src/recipes/config/default-media.config.ts

```diff
@@ -0,0 +1,31 @@
+import { MediaConfig } from '@spartacus/storefront';
```

Comment:
Let's not copy the default configs that may be imported from storefront lib. It applies also to the default cms content config.